scripts/shared/backport.sh

```diff
@@ -95,7 +95,12 @@ PULLSUBJ=$(join " " "${PULLS[@]/#/#}") # Generates something like "#12345 #56789
 declare -r PULLSUBJ
 
 echo "+++ Updating remotes..."
-git remote update "${UPSTREAM_REMOTE}" "${FORK_REMOTE}"
+if ! git remote update "${UPSTREAM_REMOTE}" "${FORK_REMOTE}"; then
```

Comment:
This has been documented at 2 places
1. In this script at L67 (which is sort of a `--help` for this script)
2. in the website (right above https://submariner.io/development/backports/#pull-requests-requiring-dependent-backports)

These are the places where a new user would go to figure out how to use `make backport`. However, I don't mind an additional check here.

Reply:
Nice, thanks @Jaanki! Yes I didn't read the docs, I just ran it and looked at the offending line. :sweat_smile: